stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -201,7 +201,7 @@ function complete_path(path::AbstractString, pos; use_envpath=false)
     end
 
     matchList = String[replace(s, r"\s" => "\\ ") for s in matches]
-    startpos = pos - lastindex(prefix) + 1 - length(matchall(r" ", prefix))
+    startpos = pos - lastindex(prefix) + 1 - count(c -> c == ' ', prefix)
```

Comment:
I hope this change is correct. AFAICT it is, but the original code is such a convoluted way of doing this that I feel like I may be missing something.

Reply:
Looks right to me, too.  Could be `equalto(' ')`.

Reply:
Yeah, I realized that right after pushing. Fixed while rebasing.